dist/s_string.ok

```diff
@@ -1415,6 +1418,7 @@ usercfg
 usr
 utf
 util
+utilise
```

Comment:
I think the American spelling of utilize already exists and should be used instead